frontends/web/src/routes/account/summary/initializeall.tsx

```diff
@@ -91,13 +91,15 @@ class InitializeAllAccounts extends Component<Props, State> {
         this.unsubscribe();
     }
 
-    public render({ t }: RenderableProps<Props>) {
+    public render({ t }: RenderableProps<Props>, { initialized }: State) {
         if (this.allInitialized()) {
             return <AccountsSummary />;
         }
         return (
             <div>
-                <Spinner text={t('accountSummary.synchronizing')}/>
+                <Spinner text={t('accountSummary.synchronizing') +
+                Object.values(initialized).filter(status => status === true).length +
+                `/${this.props.accounts.length}`}/>
```

Comment:
Rather use `render({ t, accounts }: RenderableProps<Props>, …` to be consistent.